features/FEATURE_PSA/psa_targets_in_mbed-os.md

```diff
@@ -0,0 +1,600 @@
+    to Mbed OS **MUST** be generated using the config option
+    `ConfigCoreIPC.cmake`.
+1. Ensure TF-M binary is copied to location defined by target attribute
+   `tfm_delivery_dir`
+1. Run `mbed compile -m <new target> -t <toolchain>` to build Mbed OS and run
+   regression tests
+1. Flash the regression tests binary on to the target and ensure all regression
+   tests passes
+
+*TIP*\
+Supported options:
+```console
+usage: build_tfm.py [-h]
+                    [-c {ConfigCoreIPC.cmake,ConfigRegressionIPC.cmake,ConfigPsaApiTestIPC.cmake}]
+                    [-m {ARM_MUSCA_A1,ARM_MUSCA_B1,CY8CKIT_064S2_4343W,CYESKIT_064B0S2_4343W}]
+                    [-t {ARMCLANG,GNUARM}] [-d] [-l] [--commit]
+                    [-s {CRYPTO,INITIAL_ATTESTATION,PROTECTED_STORAGE,INTERNAL_TRUSTED_STORAGE}]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c {ConfigCoreIPC.cmake,ConfigRegressionIPC.cmake,ConfigPsaApiTestIPC.cmake}, --config {ConfigCoreIPC.cmake,ConfigRegressionIPC.cmake,ConfigPsaApiTestIPC.cmake}
+                        Use the specified TF-M configuration
+  -m {ARM_MUSCA_A1,ARM_MUSCA_B1,CY8CKIT_064S2_4343W,CYESKIT_064B0S2_4343W}, --mcu {ARM_MUSCA_A1,ARM_MUSCA_B1,CY8CKIT_064S2_4343W,CYESKIT_064B0S2_4343W}
+                        Build for the given MCU
+  -t {ARMCLANG,GNUARM}, --toolchain {ARMCLANG,GNUARM}
+                        Build for the given toolchain (default is
+                        tfm_default_toolchain)
+  -d, --debug           Set build profile to debug
+  -l, --list            Print supported TF-M secure targets
+  --commit              Commit secure binaries (TF-M) and
+                        features/FEATURE_PSA/TARGET_TFM/VERSION.txt
+  -s {CRYPTO,INITIAL_ATTESTATION,PROTECTED_STORAGE,INTERNAL_TRUSTED_STORAGE}, --suite {CRYPTO,INITIAL_ATTESTATION,PROTECTED_STORAGE,INTERNAL_TRUSTED_STORAGE}
+                        Suite name for PSA API Tests
+```
+
+## Building TF-M and running PSA compliance tests
+Follow the steps below to build TF-M and compliance tests:
+
+1. Switch to the `mbed-os-tf-m-regression-tests` directory.
+1. Edit `main.cpp` to set `RUN_PSA_COMPLIANCE_TESTS` to `1` and
+  `RUN_REGRESSION_TESTS` to `0`.
+1. Run `python3 build_psa_compliance.py -m <new target> -t <toolchain> -s
+   CRYPTO`
+    * The command builds PSA API compliance tests for crypto suite
+1. Run `python3 build_tfm.py -m <new target> -t <toolchain> -c
+   configs/ConfigPsaApiTestIPC.cmake -s CRYPTO`
+    * The command builds TF-M including PSA API compliance tests
+1. Run `mbed compile -m <new target> -t <toolchain>` to build Mbed OS and run
+   PSA API compliance tests
+1. Flash the PSA API compliance tests binary on to the target and ensure all
+   PSA API compliance tests passes
+    *NOTE*\
+    If any of the PSA API compliance tests fail in TF-M example application
+    then those tests will fail in Mbed OS as well.
+
+*TIP*\
+Supported options:
+```console
```

Comment:
It would be good to state `build_psa_compliance.py -h` to see available options that avoid any updates to this section in the future if there any stales.